pkg/kev/converter/kubernetes/project_service_test.go

```diff
@@ -1022,10 +1016,19 @@ var _ = Describe("ProjectService", func() {
 			Context("when invalid policy has been provided", func() {
 				policy := "invalid-policy"
 
-				BeforeEach(func() {
-					labels = composego.Labels{
-						config.LabelWorkloadRestartPolicy: policy,
+				JustBeforeEach(func() {
```

Comment:
No sure why we started using `JustBeforeEach`?

Reply:
because we need to change the extensions and in this case the tests needs to override the creation of the project service since it can't be updated easily

Reply:
See https://github.com/appvia/kev/pull/482#discussion_r620190730